sched/job_def_serializer_test.go

```diff
@@ -1,3 +1,5 @@
+// +build property
```

Comment:
yes this.  Sometimes property tests take longer to run, so you don't always want to run them when iterating on code